### DISCUSSION TITLE
Stretch tool

### DISCUSSION THREAD

**Discussion by sxodev** [PintaProject/Pinta#762] [Ideas]
Pretty self explanatory, would work similar to the shape tool where you can stretch each corner of a selection without resizing it.

**Comment by cameronwhite** [comment]
Could you be more specific about what you mean by "stretching without resizing"?

**Reply by Lehonti** [reply]
I think it refers to expanding the area of a selection after the fact (without scalong the selection), similar to how one would do it with a crop tool

**Reply by cameronwhite** [reply]
I think that sounds like the existing `Move Selection` tool? Some feature requests like #585 would apply to make resizing the selection easier though
